Source/MediaInfo/Tag/File_Exif.cpp

```diff
@@ -1174,6 +1297,19 @@ void File_Exif::Get_IFDOffset(int8u KindOfIFD)
 //---------------------------------------------------------------------------
 void File_Exif::GetValueOffsetu(ifditem &IfdItem)
 {
+    auto GetDecimalPlaces = [](int32u denominator) -> int8u {
+        if (denominator == 1)
+            return 0;
+        int8u count{ 1 };
+        while (denominator > 10) {
+            ++count;
+            denominator /= 10;
+        }
+        if (denominator == 10)
+            return count;
+        return count + 3;
```

Comment:
```suggestion
        return count;
```

I don't see why you want more in case of no multiple of 10, with my change I have e.g.:
- 282581/2581 = 109.4851
- 282582/2581 = 109.4855

So we don't lose any precision, no need of extra digit.

Reply:
The example you give is `count + 1`.

Reply:
> The example you give is count + 1.

the +1 is done in the loop already.

Reply:
> the +1 is done in the loop already.

Right....

> Need to manage cases of more than 1 and less than 1. Altitude with more precision is not useful.

Need a way to handle all the different shutter speeds so that the original fraction can be recovered. Not all cameras use 1/x too...

Reply:
> Looks okay now? Except focal length 17/4 = 4.2 instead of 4.25

I am in favor to start with such kind of method then we tweak step by step.